libs/internal/console-driver/test-util/ng-package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "../../../../node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "../../../../dist/libs/internal/console-driver/test-util",
```

Comment:
Could you explain why we need to define the `dest` here and in the `project.json`? How does this work?